src/main/java/trivia/models/Player.java

```diff
@@ -0,0 +1,36 @@
+package trivia.models;
+
+import lombok.Data;
+
+
+@Data
+public class Player {
+
+    private long id;
+    private final String name;
+    private int place;
+    private int coins;
+
+    public Player(String name) {
+        this.name = name;
+    }
+
+    public void move(int steps) {
+        place += steps;
+    }
+
+    public void addCoins() {
+        coins += 1;
+    }
+
+    public void checkPosition(int max){
```

Comment:
misleading name: it doesn't checks anything (this sounds like it is a query with no side effects). It actually is a command, that manipulates place. 
11 is a magic number